main.go

```diff
@@ -15,7 +15,7 @@ import (
 
 func main() {
 	log.SetOutput(os.Stdout)
-	cfg := config.NewConfig()
+	cfg := config.NewConfig("config")
```

Comment:
Do you agree with this approach? To leave `"config"` as the default?

Reply:
Yes.